exercises/armstrong-numbers/armstrong_numbers_test.cpp

```diff
@@ -0,0 +1,52 @@
+#include "test/catch.hpp"
+#include "armstrong_numbers.h"
+
+// Armstrong-number exercise test case data version 1.1.0
+TEST_CASE("zero_is_an_armstrong_number")
+{
+    REQUIRE(armstrong_numbers::is_armstrong_number(0));
+}
+
+#if defined(EXERCISM_RUN_ALL_TESTS)
+
+TEST_CASE("single_digit_numbers_are_armstrong_numbers")
+{
+    REQUIRE(armstrong_numbers::is_armstrong_number(5));
+}
+
+TEST_CASE("there_are_no_2_digit_armstrong_numbers")
+{
+    REQUIRE(!armstrong_numbers::is_armstrong_number(10));
```

Comment:
I just realized, Catch has a `REQUIRE_FALSE` macro that would probably make these test cases clearer even though we haven't used it in other test suites. Maybe it's worth it to use that macro instead for clarity?